engine/gamesys/src/gamesys/components/comp_collection_proxy.cpp

```diff
@@ -1043,4 +1093,51 @@ namespace dmGameSystem
      * end
      * ```
      */
+
+    /*# changes the collection for a collection proxy.
+     * 
+     * The collection should be loaded by the collection proxy.
+     * Setting the collection to "nil" will revert it back to the original collection.
+     * 
+     * The collection proxy shouldn't be loaded and should have the 'Exclude' checkbox checked.
+     * This functionality is designed to simplify the management of Live Update resources.
+     *
+     * @name collectionproxy.set_collection
+     * @param [url] [type:string|hash|url] the collection proxy component
+     * @param [prototype] [type:string|nil] the path to the new collection, or `nil`
```

Comment:
what does nil mean here, that you cant load it?

Reply:
>Setting the collection to "nil" will revert it back to the original collection.